lib/klaviyo/lists.rb

```diff
@@ -0,0 +1,63 @@
+## base lists, list, add_members_to_list from https://github.com/beastgroup/ruby-klaviyo/commit/91fe4e03c9c1c8fc9e454c3927b010f9e4ccc2c9
+
+require 'rest-client'
+
+module Klaviyo
+  module Lists
+
+    def lists
+      _RESOURCE = 'api/v1/lists'
```

Comment:
Apologize for the delay on reviewing here.  We have just released the v2 lists api endpoint and would prefer to use this: https://www.klaviyo.com/docs/api/v2/lists

Reply:
Sorry, but we've moved on from this library.  Waiting 5 months to respond and then asking a submitter to make changes to update is ridiculous.

Reply:
agreed and thank you for your feedback.